<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口通信软件</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/dianbiao.jpg</normaloff>:/img/dianbiao.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QWidget" name="widget_9" native="true">
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>100</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="message"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QRadioButton" name="hexSend">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Hex发送</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="charSend">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>字符</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="btnSend">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_8" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="1">
          <widget class="QComboBox" name="stopBits">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>波特率：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>数据位：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="portName">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>停止位：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>校验位：</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QRadioButton" name="charRecv">
           <property name="text">
            <string>字符显示</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="openPort">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>打开串口</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>串口 ：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="dataBits">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="parityBits">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>偶校验</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>无</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>奇校验</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QRadioButton" name="hexRecv">
           <property name="text">
            <string>Hex显示</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="baudRate">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>28800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="btnClear">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>输入流水号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="DI3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>发送报文</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="recvData_2">
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="open_xiebo">
             <property name="text">
              <string>打开谐波监测</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="close_xiebo">
             <property name="text">
              <string>关闭谐波监测</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>接收信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="recvData">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuasdfasdf">
    <property name="title">
     <string>1</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuasdfasdf"/>
  </widget>
  <action name="github">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/github.ico</normaloff>:/img/github.ico</iconset>
   </property>
   <property name="text">
    <string>github</string>
   </property>
  </action>
  <action name="readMe">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/help.ico</normaloff>:/img/help.ico</iconset>
   </property>
   <property name="text">
    <string>README</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
